src/main/java/com/iyzipay/model/PaymentCheckoutFormMetaDataKey.java

```diff
@@ -0,0 +1,5 @@
+package com.iyzipay.model;
+
+public enum PaymentCheckoutFormMetaDataKey {
+    DEBIT_CARD_ALLOWED
```

Comment:
We need to re-structure this enum with constructed enum which has string values. In this case;
DEBIT_CARD_ALLOWED("debitCardAllowed")